pkg/database/realm.go

```diff
@@ -874,21 +874,6 @@ func (r *Realm) ListMemberships(db *Database, p *pagination.PageParams, scopes .
 	return memberships, paginator, nil
 }
 
-// FindMembership finds the membership with the given id, if it exists.
-func (r *Realm) FindMembership(db *Database, userID interface{}) (*Membership, error) {
```

Comment:
`user.FindMembership(db, realmId)` is in use, probably doesn't make sense to have two